enterprise/cmd/frontend/internal/auth/gitlaboauth/login.go

```diff
@@ -24,6 +24,8 @@ func CallbackHandler(config *oauth2.Config, success, failure http.Handler) http.
 }
 
 func gitlabHandler(config *oauth2.Config, success, failure http.Handler) http.Handler {
+	logger := log.Scoped("gitlab-handler", "Gitlab Handler")
```

Comment:
```suggestion
	logger := log.Scoped("gitlabHandler", "GitLab oAuth handler")
```

> Scope names should be CamelCase or camelCase, and the scope description should follow [the same conventions as a log message](https://docs.sourcegraph.com/dev/how-to/add_logging#writing-log-messages).

https://docs.sourcegraph.com/dev/how-to/add_logging#scoped-loggers

Reply:
"oAuth" should be "OAuth" :)